content/authors/rhett-skelton/_index.md

```diff
@@ -27,20 +27,20 @@ agency_full_name: ""
 agency: ""
 
 # Tell us where you live and work [e.g. 'New York City' or 'Portland, OR']
-location: ""
+location: "Washington, DC"
 
 # A GitHub account will allow you to edit pages on Digital.gov. Also, the image used in your GitHub account can be used to populate your digital.gov profile photo. Learn more about getting a Github account at [URL]
 github: "RhettSkelton"
 
 # Profile Photo
 # See [URL] for a full list of profile photo options
-profile_photo: "github"
+profile_photo: ""
 
 # [e.g., Digital_Gov]
 twitter: ""
 facebook: ""
 instagram: ""
-linkedin: ""
+linkedin: "https://www.linkedin.com/in/rhettskelton/"
```

Comment:
I tested this link in the preview and it didn't work. Looks like you just want to enter your username and not the whole URL.